code/game/gamemodes/blob/blobs/core.dm

```diff
@@ -32,11 +32,7 @@
 
 /obj/structure/blob/core/update_icon()
 	cut_overlays()
-	color = null
-	var/image/I = new('icons/mob/blob.dmi', "blob")
-	if(overmind)
-		I.color = overmind.blob_reagent_datum.color
-	add_overlay(I)
+	..()
 	var/image/C = new('icons/mob/blob.dmi', "blob_core_overlay")
 	add_overlay(C)
```

Comment:
Image might need RESET_COLOR or something so the overlay isn't colored along with the blob it's on